docs/src/modules/spring/pages/views.adoc

```diff
@@ -1 +1,167 @@
+
+This example assumes a Customer equal to the previous example and an Event Sourced Entity that uses this Customer and it is in charge of producing the events that update the View. These events are defined as subtypes of the class `CustomerEvent` following standard https://github.com/FasterXML/jackson-annotations#handling-polymorphic-types[Jackson notation] like this: 
+
+.src/main/java/customer/api/CustomerEvent.java
+[source,java]
+----
+include::example$spring-eventsourced-customer-registry/src/main/java/customer/api/CustomerEvent.java[tags=class]
+----
+<1> Includes the logical type name (defined in implementation classes) as type 
+<2> Required for deserialization and to define the logical type name 
+
+Adding `@JsonTypeInfo` is needed so that Jackson can read back correctly the Object type when deserializing. By using `name`, you set its value as the fully qualified message. This is called a `logical type name` and it allows you to refactor the class to another package after you have serialized messages of this type. Otherwise, after such a refactor the objects serialized with the old package won't be recognized as the same type. 
+
+
+=== Define the View to consume events
+
+The definition of the view for an Event Sourced Entity is the same as for a Value Entity. However, in this example, the subscription is at the method level rather than the type level. The advantage of this approach is that you can create multiple methods to handle different events. It is recommended you add a view ID to your view. 
+
+Every time an event is processed by the view, the state of the view can be updated. You can do this with the `.updateState` method, which is available through the `effects()` API. Here you can see how the View is updated with a new name: 
+
+ .src/main/java/customer/view/CustomerByNameView.class
+[source,java]
+----
+include::example$spring-eventsourced-customer-registry/src/main/java/customer/view/CustomerByNameView.java[tags=class]
+----
+<1> setting view ID 
+<2> updating the state of the view with the new name 
+
+An Event Sourced entity can emit many types of events. You need to define a method for each event type. They return an `UpdateEffect`, which describes next processing actions, such as updating the view state.
+
+
+
+
+See <<#query>> for more examples of valid query syntax.
+
+== Ignoring events
+
+When consuming events, each event must be matched by a View service method. In case your View is only interested in certain events, you may declare a catch-all method to receive the `sealed interface` and don't update the View. If an event type is not handled, the View will fail. Views are designed to restart, but since the handler is missing, it will fail again. Therefore, it's important to define methods for all events or define a catch-all method in case you want to discard some events.
```

Comment:
I elaborated on the `ignoreUnknown` explanation but I didn't promote nor add type level annotations explanation. Let me see if we have an example already.

Reply:
I'd like to discuss this last point if possible.

Reply:
Elaborated on it and linked to type-level annotations in actions which has an example explaining exactly the same. I have chosen to use the link because I think is more didactic. It doesn't add more info than the reader needs to understand views and brings the idea that Actions work in the same way when ignoring events.